screens/CreateGame.js

```diff
@@ -27,30 +30,61 @@ function CreateGame({ navigation }) {
   const [chosenDate, setChosenDate] = useState(new Date());
   const [showError, setShowError] = useState(false);
   const [errorMessage, setErrorMessage] = useState("");
+  const [locations, setLocations] = useState([]);
+  const [selectedPlayers, setSelected] = useState([]);
+
+  //get all locations to populate the location picker
+  async function getLocations() {
+    setLoading(true);
+    try {
+      const allLocations = await DataStore.query(Location);
+      //map the name to an array
+      const formatLocations = allLocations.map(({ name: value }) => ({
+        value,
+        label: value,
+      }));
+      setLocations(formatLocations);
+    } catch (error) {
+      //if locations don't load for some reason then we should probably display an error message and route back
+      console.log(error.message);
+    }
+    setLoading(false);
+  }
+
+  async function getPlayers() {
+    setLoading(true);
+    let toInvite = [];
+    try {
+      const allPlayers = await DataStore.query(Player, (p) => p.email.ne(thisPlayer.email));
+      allPlayers.map((element) => {
+        toInvite.push({label: element.name, value: element.id});
+      });
+      console.log("to invite: ", toInvite);
+      setToInvite(toInvite);
+    } catch (error) {
+      console.log("Error occurred in getting all players: " + error.message);
```

Comment:
Why not set error message inside catch blocks? There already is one in this component, so use it.